frontend/public/tinymce/langs/zh_TW.js

```diff
@@ -0,0 +1,419 @@
+"Color Picker": "\u9078\u8272\u5668",
+"R": "\u7d05",
+"G": "\u7da0",
+"B": "\u85cd",
+"Left to right": "\u5f9e\u5de6\u5230\u53f3",
+"Right to left": "\u5f9e\u53f3\u5230\u5de6",
+"Emoticons...": "\u8868\u60c5\u7b26\u865f\u2026",
+"Metadata and Document Properties": "\u5f8c\u8a2d\u8cc7\u6599\u8207\u6587\u4ef6\u5c6c\u6027",
+"Title": "\u6a19\u984c",
+"Keywords": "\u95dc\u9375\u5b57",
+"Description": "\u63cf\u8ff0",
+"Robots": "\u6a5f\u5668\u4eba",
+"Author": "\u4f5c\u8005",
+"Encoding": "\u7de8\u78bc",
+"Fullscreen": "\u5168\u87a2\u5e55",
+"Action": "\u52d5\u4f5c",
+"Shortcut": "\u5feb\u901f\u9375",
+"Help": "\u5e6b\u52a9",
+"Address": "\u5730\u5740",
+"Focus to menubar": "\u8df3\u81f3\u9078\u55ae\u5217",
+"Focus to toolbar": "\u8df3\u81f3\u5de5\u5177\u5217",
+"Focus to element path": "\u8df3\u81f3HTML\u5143\u7d20\u5217",
+"Focus to contextual toolbar": "\u8df3\u81f3\u5feb\u6377\u9078\u55ae",
+"Insert link (if link plugin activated)": "\u65b0\u589e\u6377\u5f91 (\u6377\u5f91\u5916\u639b\u555f\u7528\u6642)",
+"Save (if save plugin activated)": "\u5132\u5b58 (\u5132\u5b58\u5916\u639b\u555f\u7528\u6642)",
+"Find (if searchreplace plugin activated)": "\u5c0b\u627e (\u5c0b\u627e\u53d6\u4ee3\u5916\u639b\u555f\u7528\u6642)",
+"Plugins installed ({0}):": "({0}) \u500b\u5916\u639b\u5df2\u5b89\u88dd\uff1a",
+"Premium plugins:": "\u52a0\u503c\u5916\u639b\uff1a",
+"Learn more...": "\u4e86\u89e3\u66f4\u591a...",
+"You are using {0}": "\u60a8\u6b63\u5728\u4f7f\u7528 {0}",
+"Plugins": "\u5916\u639b",
+"Handy Shortcuts": "\u5feb\u901f\u9375",
+"Horizontal line": "\u6c34\u5e73\u7dda",
+"Insert\/edit image": "\u63d2\u5165\/\u7de8\u8f2f \u5716\u7247",
+"Image description": "\u5716\u7247\u63cf\u8ff0",
+"Source": "\u5716\u7247\u7db2\u5740",
+"Dimensions": "\u5c3a\u5bf8",
+"Constrain proportions": "\u7b49\u6bd4\u4f8b\u7e2e\u653e",
+"General": "\u4e00\u822c",
+"Advanced": "\u9032\u968e",
+"Style": "\u6a23\u5f0f",
+"Vertical space": "\u9ad8\u5ea6",
+"Horizontal space": "\u5bec\u5ea6",
+"Border": "\u908a\u6846",
+"Insert image": "\u63d2\u5165\u5716\u7247",
+"Image...": "\u5716\u7247......",
+"Image list": "\u5716\u7247\u6e05\u55ae",
+"Rotate counterclockwise": "\u9006\u6642\u91dd\u65cb\u8f49",
+"Rotate clockwise": "\u9806\u6642\u91dd\u65cb\u8f49",
+"Flip vertically": "\u5782\u76f4\u7ffb\u8f49",
+"Flip horizontally": "\u6c34\u5e73\u7ffb\u8f49",
+"Edit image": "\u7de8\u8f2f\u5716\u7247",
+"Image options": "\u5716\u7247\u9078\u9805",
+"Zoom in": "\u653e\u5927",
+"Zoom out": "\u7e2e\u5c0f",
+"Crop": "\u88c1\u526a",
+"Resize": "\u8abf\u6574\u5927\u5c0f",
+"Orientation": "\u65b9\u5411",
+"Brightness": "\u4eae\u5ea6",
+"Sharpen": "\u92b3\u5316",
+"Contrast": "\u5c0d\u6bd4",
+"Color levels": "\u984f\u8272\u5c64\u6b21",
+"Gamma": "\u4f3d\u99ac\u503c",
+"Invert": "\u53cd\u8f49",
+"Apply": "\u61c9\u7528",
+"Back": "\u5f8c\u9000",
+"Insert date\/time": "\u63d2\u5165 \u65e5\u671f\/\u6642\u9593",
+"Date\/time": "\u65e5\u671f\/\u6642\u9593",
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 0.](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7687804)
```suggestion
  "Date\/time": "\u65e5\u671f\/\u6642\u9593",
```